### ISSUE TITLE
Windows Basic style in also available in Windows 8-10-11

### ISSUE THREAD

**Issue by OrthodoxWindows** [nptr/msstyleEditor#74]
The "Basic" windows frame, which appears when DWM is disabled (It is possible to disable DWM in Windows 8 and higher, but this is no longer usable without bigger issues from Windows 10 1703), can be activated in Windows 8 and higher **without disable DWM**, thanks to this program : https://github.com/Ingan121/BasicThemer2

This gives numerous possibility of creation, because the window class is more flexible than the DWM Window class.
Which is also interesting is that it meant that Windows 8 and upper contain seven window frame style fonctionnal without third-party visual styles. So we have, Aero, Aero Basic, Aero Lite, Aero Lite Basic, Aero Lite High contrast, Aero Lite Basic High contrast, and Classic.

**Comment by nptr** [comment]
Yep, look like you can just tell DWM to not use visual styles for a specific window.
I am not sure what you want me to do with this information.

**Comment by OrthodoxWindows** [comment]
In your wiki, in the "Window Frame" chapter, you mention "two different locations for the window frames and other components, the Window class and the DWMWindow class", and to assert you which is not necessary to modify the values contained in the "Window" class, because it does not do what the user sees every day.

With BasicThemer2, it is possible to use the "Window class" every day (on almost all programs), without disable DWM.
My request is therefore on the wiki : would it be possible to correct the affirmation according to which it is not necessary to worry about the "Window" class ? Because it's no longer true.

If you wish, I can try writing documentation on "Window" class, but of this case, it will be necessary to verify the relevance of what I write, because I am not good in English (it is not my mother tongue).

**Comment by nptr** [comment]
I'll clarify that 👍

**Comment by nptr** [comment]
I changed it to be not as asserting as before. I decided I'll not document the `Window` class however but take your offer.
Write whatever you feel is important. I'll review it in any case, although I am not a native speaker myself.

----

> [..] that Windows 8 and upper contain seven window frame style fonctionnal without third-party visual styles. So we have, Aero, Aero Basic, Aero Lite, Aero Lite Basic, Aero Lite High contrast, Aero Lite Basic High contrast, and Classic.

HighContrast and DarkMode is such a mess btw. It looks like visual styles support multiple variants, but Microsoft only ever uses the "normal" variant. Why they added dozen new classes and properties, instead of creating a new variant is beyond me. Heck, they could have just made a new visual style like they did with aerolite.

**Comment by OrthodoxWindows** [comment]
_"I changed it to be not as asserting as before. I decided I'll not document the Window class however but take your offer.
Write whatever you feel is important. I'll review it in any case, although I am not a native speaker myself."_

Thank you for rectification. For the documentation, I will do it soon, when I have the time. I also have to complete the documentation on the file explorer.

_"HighContrast and DarkMode is such a mess btw. It looks like visual styles support multiple variants, but Microsoft only ever uses the "normal" variant. Why they added dozen new classes and properties, instead of creating a new variant is beyond me. Heck, they could have just made a new visual style like they did with aerolite."_

I totally agree ; i made myself the same remark by exploring the structure of the aero and aerolite visual style. In Windows XP, that's what Microsoft did: Add additional .msstyle.
For the DarkMode, i plan to replace the light controls of Aero by the dark so that all programs supports the DarkMode.
To come back about the Window class, what I appreciate in the basic WindowsFrame, it is its flexibility about the corners (it is possible to choose the shape), and the ability to support the square buttons (and not only retangular). On the other hand, it is of course not possible to obtain transparency is animation :).
